code401challenges/src/test/java/code401challenges/ArrayShiftTest.java

```diff
@@ -0,0 +1,15 @@
+package code401challenges;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ArrayShiftTest {
+
```

Comment:
full points today, but in the future id like to see more tests. for example: test the edge cases, what would happen if it was a 0 length array, what happens in even vs odd array length?